src/MovieCard/MovieCard.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import MovieCard from './MovieCard.js';
+
+describe('MovieCard', () => {
+  it('should render a movie card', () => {
+    render(
+      <MovieCard
+        key={1}
+        avgRating={3}
+        title="Titanic"
+        photo="image path"
+        releaseDate={"2020-09-29"}
+      />);
+    expect(screen.getByText('Average Rating: 3')).toBeInTheDocument();
+    expect(screen.getByText('Titanic')).toBeInTheDocument();
+    expect(screen.getByAltText('Titanic movie poster')).toBeInTheDocument();
```

Comment:
Nice way to test that the image is showing up! I wonder if there is a way to test the actual image? Might need to do more research on that and come back to it if we find anything